webapp/packages/core-blocks/src/Tree/TreeNode/TreeNodeSelect.m.css

```diff
@@ -0,0 +1,10 @@
+.treeNodeSelect {
+  &.big {
+    margin: 0;
+  }
```

Comment:
we can probably move all `big` related styles to the separate registry and apply it to the tree if needed with `SContext`